Makefile

```
@@ -58,7 +58,7 @@ export EIGEN_LDFLAGS :=
 ifeq ($(wildcard /usr/include/boost/version.hpp),)
 NEED_BOOST := true
 else
-NEED_BOOST := $(shell (( $$(cat /usr/include/boost/version.hpp | grep '\#define BOOST_VERSION\>' | awk '{ print $$3 }') > 106300 )) || echo true)
+NEED_BOOST := $(shell test $$(cat /usr/include/boost/version.hpp | grep '\#define BOOST_VERSION\>' | awk '{ print $$3 }') -ge 106300 || echo true)
```

This should actually be
```suggestion
NEED_BOOST := $(shell test $$(cat /usr/include/boost/version.hpp | grep '\#define BOOST_VERSION\>' | awk '{ print $$3 }') -gt 106300 || echo true)
```
to preserve the earlier intent.